tests/Unit/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/Test_InitializeElementFacesGridCoordinates.cpp

```diff
@@ -125,6 +125,7 @@ SPECTRE_TEST_CASE(
         std::array<double, 2>{{0.1, 0.2}},
         30.,
         50.,
+        13.0,
```

Comment:
Make this `std::nullopt` so it's guaranteed to be the separation even if the x-coords of this test change. Same for the rest of these tests that only use the BCO domain